### PR TITLE
CVE-2005-2700

### PR DESCRIPTION
**Pull request by NateTheGr8Falk**
My answers for this CVE.

### CODE REVIEW

File: cves/CVE-2005-2700.yml [hunk 1]
```diff
@@ -199,4 +208,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This mistake was a design a mistake because it failed to verify client certificates during
```
Comment (ben-dow) [inline cves/CVE-2005-2700.yml:212]: How could the mistake be prevented in the future?

**Comment by ben-dow** [comment]
I would like to give 4 upvotes

File: cves/CVE-2005-2700.yml [hunk 2]
```diff
@@ -154,37 +163,37 @@ lessons:
...
     note:
   serial_killer:
-    applies:
+    applies: false
     note:
   environment_variables:
-    applies:
-    note:
+    applies: true
+    note: The bug is caused by a global configuration of the apache server.
```
Comment (jcotton42) [inline cves/CVE-2005-2700.yml]: I'd always thought that the environment variables bit was referring to storing sensitive data in them, which may be visible to other processes

**Comment by jcotton42** [comment]
I'd like to give this CVE 2 upvotes.

**Pull request merged**